recipes/_environment.rb

```diff
@@ -110,6 +110,10 @@
   end
 else
 
+  if aix?
+    omnibus_env['PATH'].unshift('/opt/IBM/xlC/13.1.0/bin:/opt/IBM/xlC/13.1.0/bin')
+  end
+
```

Comment:
Why does `/opt/IBM/xlC/13.1.0/bin` have to come first in the `PATH`? Even if that is so you could do:

```ruby
omnibus_env['PATH'] << "/opt/#{node['omnibus']['toolchain_name']}/embedded/bin" if omnibus_toolchain_enabled?
omnibus_env['PATH'] << '/usr/local/bin'

if aix?
  omnibus_env['PATH'].unshift('/opt/IBM/xlC/13.1.0/bin')
elsif solaris_10?
  omnibus_env['PATH'] << '/usr/sfw/bin'
  omnibus_env['PATH'] << '/usr/ccs/bin'
end
```